packages/tldraw/src/lib/ui/components/CommandBar/DefaultCommandBar.tsx

```diff
@@ -0,0 +1,223 @@
+import { useEditor } from '@tldraw/editor'
+import classNames from 'classnames'
+import { useCallback, useState } from 'react'
+import { TLUiActionItem, unwrapLabel, useActions } from '../../context/actions'
+import { useCommandBarActions } from '../../hooks/useCommandBarActions'
+import { useMenuIsOpen } from '../../hooks/useMenuIsOpen'
+import { TLUiTranslationKey } from '../../hooks/useTranslation/TLUiTranslationKey'
+import { useTranslation } from '../../hooks/useTranslation/useTranslation'
+import { TldrawUiButton } from '../primitives/Button/TldrawUiButton'
+import { TldrawUiButtonLabel } from '../primitives/Button/TldrawUiButtonLabel'
+import { TldrawUiKbd } from '../primitives/TldrawUiKbd'
+
+export const COMMAND_BAR_ID = 'command bar'
+const MAX_ITEMS = 6
```

Comment:
can the command bar just be scrollable? feels maybe limiting to just have it be the top 6 items

Reply:
I guess so, but didn't really feel the need for it. Once you enter a few chars you quite quickly narrow the actions down to less than 6. I guess it might be more useful if we added fuzzy search which would match more actions.